src/frontend/src/app/AppPublic.tsx

```diff
@@ -43,14 +43,11 @@ const AppPublic: React.FC = () => {
         (prevArgs: string, pathPart: string, idx: number): string => `${prevArgs}&value${idx + 1}=${pathPart}`,
         `?page=${redirectPathParts[0]}`
       );
-    const redirectQueryArgs =
-      redirectPathQueryArgs + (history.location.search ? `&${history.location.search.slice(1)}` : '');
 
     return (
       <Redirect
         to={{
           pathname: routes.LOGIN,
-          search: redirectQueryArgs,
```

Comment:
We want to keep this because it embeds the current page into the login URL so after logging in we can get back to the current page.